From 0f3c4c8f00558fcdf7bf4367125a1f0e6081f691 Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Tue, 20 Aug 2024 00:41:33 +0000
Subject: [PATCH] remove fields

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 platform/linuxbsd/wayland/wayland_thread.h | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/platform/linuxbsd/wayland/wayland_thread.h b/platform/linuxbsd/wayland/wayland_thread.h
index 775ca71346..2752d203b2 100644
--- a/platform/linuxbsd/wayland/wayland_thread.h
+++ b/platform/linuxbsd/wayland/wayland_thread.h
@@ -664,8 +664,6 @@ private:
 	static constexpr struct wl_surface_listener wl_surface_listener = {
 		.enter = _wl_surface_on_enter,
 		.leave = _wl_surface_on_leave,
-		.preferred_buffer_scale = _wl_surface_on_preferred_buffer_scale,
-		.preferred_buffer_transform = _wl_surface_on_preferred_buffer_transform,
 	};
 
 	static constexpr struct wl_callback_listener frame_wl_callback_listener {
@@ -701,7 +699,6 @@ private:
 		.axis_stop = _wl_pointer_on_axis_stop,
 		.axis_discrete = _wl_pointer_on_axis_discrete,
 		.axis_value120 = _wl_pointer_on_axis_value120,
-		.axis_relative_direction = _wl_pointer_on_axis_relative_direction,
 	};
 
 	static constexpr struct wl_keyboard_listener wl_keyboard_listener = {
-- 
2.46.0

