From 4ff5ebf00d258d22a2f0ad6a2b5bc7a35f7cd77e Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Fri, 23 Aug 2024 21:00:03 +0000
Subject: [PATCH 2/2] enable clang

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 platform/linuxbsd/detect.py | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/platform/linuxbsd/detect.py b/platform/linuxbsd/detect.py
index d1de760f34..04f4662ed2 100644
--- a/platform/linuxbsd/detect.py
+++ b/platform/linuxbsd/detect.py
@@ -110,9 +110,6 @@ def configure(env: "SConsEnvironment"):
         env["use_llvm"] = True
 
     if env["use_llvm"]:
-        if "clang++" not in os.path.basename(env["CXX"]):
-            env["CC"] = "clang"
-            env["CXX"] = "clang++"
         env.extra_suffix = ".llvm" + env.extra_suffix
 
     if env["linker"] != "default":
-- 
2.46.0

