From 3d0b3397e92a3a8d67842fb15a1c9cedc7ed5ccb Mon Sep 17 00:00:00 2001
From: Joel Winarske <joel.winarske@gmail.com>
Date: Tue, 21 May 2024 07:03:32 -0700
Subject: [PATCH] Add AS AR RANLIB RC

Signed-off-by: Joel Winarske <joel.winarske@gmail.com>
---
 SConstruct | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/SConstruct b/SConstruct
index 0afdb07b35..62c872f122 100644
--- a/SConstruct
+++ b/SConstruct
@@ -259,6 +259,10 @@ opts.Add(BoolVariable("builtin_zstd", "Use the built-in Zstd library", True))
 opts.Add("CXX", "C++ compiler")
 opts.Add("CC", "C compiler")
 opts.Add("LINK", "Linker")
+opts.Add("AS", "Assembler binary")
+opts.Add("AR", "Archiver binary")
+opts.Add("RANLIB", "Ranlib binary")
+opts.Add("RC", "Resource compiler binary")
 opts.Add("CCFLAGS", "Custom flags for both the C and C++ compilers")
 opts.Add("CFLAGS", "Custom flags for the C compiler")
 opts.Add("CXXFLAGS", "Custom flags for the C++ compiler")
-- 
2.45.0

